scripts/checkRulesCoverage

```diff
@@ -57,11 +57,12 @@ function foundRule(r) {
 		delete unmatched[r]
 	}
 }
-
-const grepCmd = execSync( `grep -r . ${folder}`, { maxBuffer: 3e6, encoding:'ascii'})
+cmd=`grep -E -r '\\b\\w{2,4}-\\d{2,4}' ${folder}`
```

Comment:
@drortirosh This file contains a lot of magic. Please add comments to explain what is happening throughout this file. Thank you!